vault/data_source_kv_subkeys_v2_test.go

```diff
@@ -27,6 +27,9 @@ func TestDataSourceKVSubkeys(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, fmt.Sprintf("%s/subkeys/%s", mount, secretPath)),
 					resource.TestCheckResourceAttrSet(resourceName, consts.FieldDataJSON),
+					resource.TestCheckResourceAttr(resourceName, "data.zip", "null"),
```

Comment:
Probably want to check the length of `data.%`

Reply:
Added in latest commit